### ISSUE TITLE
Issue loading existing RoBERTa model

### ISSUE THREAD

**Issue by sarahwie** [pytorch/fairseq#1318]
Bug appears to stem from 5a2f76ede.
```
import torch
roberta = torch.hub.load('pytorch/fairseq', 'roberta.large')
```
produces stacktrace:
```
RuntimeError: 
unexpected expression in variable initialization of for loop:
@torch.jit.script
def script_skip_tensor_list(x: List[Tensor], mask):
    res = [xi[mask] if xi.size(0) == mask.size(0) else xi[:, mask] for xi in x]
    outputs = []
    for i, t in enumerate(res):
        ~~~~ <--- HERE
        if t.numel() != 0:
            outputs.append(t)
        else:
            outputs.append(x[i])
    return outputs
```

System specs:
```
>>> sys.version
'3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) \n[GCC 7.3.0]'
>>> torch.__version__
'1.1.0'
```

**Comment by sarahwie** [comment]
Also occurs from 
```
from fairseq.models.roberta import RobertaModel
roberta = RobertaModel.from_pretrained('/hg191/swiegreffe6/roberta.large', checkpoint_file='model.pt')
```

**Comment by sarahwie** [comment]
Resolved by upgrading to PyTorch >= 1.2.0 (as specified in the README 🤦‍♀️).
